array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,12 @@
+/* exported initial */
+function initial(array) {
+  var initialArray = [];
+  for (var i = 0; i < array.length - 1; i++) {
+    if (i === array.length || i === array.length - 1) {
+      break;
+    } else {
+      initialArray.push(array[i]);
+    }
```

Comment:
You can remove this conditional logic, since you are already stopping your loop at `array.length - 1`.